### ISSUE TITLE
Multiple images upload

### ISSUE THREAD

**Issue by anaibol** [cloudinary/cloudinary_npm#99]
Hi, is it possible to upload multiple images at the same time?

**Comment by anaibol** [comment]
Anyone?

**Comment by roeeba** [comment]
While there is currently no straightforward way to achieve this, you can write a short script which goes through your images and uploads them one by one to your Cloudinary account. For more information: 
http://support.cloudinary.com/entries/23684543-How-can-I-bulk-upload-my-images- (note that you can use concurrent uploads, please keep it up to 50 uploads in parallel).

Alternatively, you can use Cloudinary powerful automatic-migration tool. 
For more information: 
http://cloudinary.com/blog/how_to_automatically_migrate_all_your_images_to_the_cloud.

**Comment by roeeba** [comment]
Closing this issue due to the time elapsed. Please feel free to either re-open the issue, contact our support at http://support.cloudinary.com or create a new ticket if you have any additional issues.
